xapian-letor/scorer/err_score.cc

```diff
@@ -59,7 +53,12 @@ ERRScore::score(const std::vector<FeatureVector> & fvv) const
 	return 0;
     }
     size_t length = fvv.size();
-    FeatureVector max_label = *std::max_element(fvv.begin(), fvv.end(), comp);
+    FeatureVector max_label = *max_element(fvv.begin(), fvv.end(),
+					   [](FeatureVector x,
+					   FeatureVector y) {
```

Comment:
We don't have an explicit style rules for lamba functions currently, so I think we should apply the rules for functions unless there's a good reason why they don't make sense, which would mean aligning `FeatureVector y` directly under `FeatureVector x`.  I think that makes sense here, and is more readable (at a glance, it currently looks like `FeatureVector y` is a parameter of the `max_element()` invocation).